### PR TITLE
Improving the Insert columns/rows dialog

### LINKED ISSUE #5124: Small bug on Insert Columns/Rows Dialog
**Describe the bug**
I've noticed that **Ok** is enabled eve when there is no data loaded. This is the case for both options, that is, **Insert Columns** and **Insert Rows**.

**Expected behaviour**
I think **Ok** should be enabled only when there is a data frame loaded, otherwise where will the inserting happen?

**Screenshots**
image

### PR DESCRIPTION
**Pull request by Muthenya**
Fixes #5124
@africanmathsinitiative/developers this is ready for review.

### CODE REVIEW

**Comment by dannyparsons** [comment]
@africanmathsinitiative/developers please review this, the changes should be minimal.

**Comment by Muthenya** [comment]
@Wycklife kindly review this.

File: instat/dlgInsertColumn.vb [hunk 1]
```diff
@@ -187,29 +180,26 @@ Public Class dlgInsertColumn
...
+    Private Sub ucrReceiverColumnsToInsert_ControlContentsChanged(ucrChangedControl As ucrCore) Handles ucrReceiverColumnsToInsert.ControlContentsChanged, ucrPnlColumnsOrRows.ControlContentsChanged, ucrPnlBeforeAfter.ControlContentsChanged, ucrPnlInsertColumns.ControlContentsChanged, ucrInputPrefixForNewColumn.ControlContentsChanged, ucrInputDefaultValue.ControlContentsChanged, ucrInputBeforeAfter.ControlContentsChanged, ucrNudNumberOfRows.ControlContentsChanged, ucrNudStartRow.ControlContentsChanged, ucrNudNumberOfColumns.ControlContentsChanged, ucrSelectorInsertColumns.ControlContentsChanged
         TestOKEnabled()
     End Sub
 
+    Private Sub ReopenDialog()
```
Comment (dannyparsons) [inline instat/dlgInsertColumn.vb]: Can this just be renamed now that it is not a method that's called on reopen.

Comment (Muthenya) [reply instat/dlgInsertColumn.vb]: It's still required on reopen. I haven't removed it from its initial position in the load event.

Comment (dannyparsons) [reply instat/dlgInsertColumn.vb]: Ok that's fine then

**Pull request merged**